src/pages/index.tsx

```diff
@@ -66,41 +67,49 @@ function Page(props: Props) {
           imageSrc="https://storeframework.vtexassets.com/arquivos/ids/190897/Photo.jpg"
           imageAlt="Quest 2 Controller on a table"
         />
-      </section>
+      </Section>
 
-      <section className="page__section">
+      <Section>
         <IncentivesHeader />
-      </section>
+      </Section>
 
-      <section className="page__section page__section-shelf / grid-section">
-        <h2 className="title-section / grid-content">Most Wanted</h2>
-        <div className="page__section-content">
-          <ProductShelf products={products?.slice(0, 5)} />
-        </div>
-      </section>
+      <Section>
```

Comment:
I'm not sure if the `Section` wrapper is the best approach since we are increasing the number of DOM nodes with `section`>`section`. Maybe we could transform the `Section.scss` into a sass `@mixin` and `@include` it elsewhere? Just thinking out loud, though.
![OJl3rqu38P](https://user-images.githubusercontent.com/11325562/156803048-1b46aece-5337-45be-8220-a3573d3ade50.gif)

Reply:
Hmm. How about receiving the extra classes on the `<Section>` component? 

For example:

```
<Section  classes="grid-section grid-content">
    <h2 className="title-section">Just Arrived</h2>
    <div className="page__section-content">
        <ProductTiles products={products?.slice(5, 8)} />
    </div>
 </Section>
```

Reply:
Nice, now we have the same number of dom nodes :)